server/runtime/src/main/java/org/infinispan/server/logging/Log.java

```diff
@@ -22,8 +22,8 @@
    void serverStarting(String name);
 
    @LogMessage(level = Logger.Level.INFO)
-   @Message(value = "%s Server %s started in %dms", id = 80001)
-   void serverStarted(String name, String version, long ms);
+   @Message(value = "%s Server %s (%s) started in %dms", id = 80001)
+   void serverStarted(String name, String version, String codename, long ms);
```

Comment:
I'm not sure how useful is to have the codename...

Reply:
The idea is to use the codename to have the upstream name + version